taskcluster/l10n_taskgraph/update.py

```diff
@@ -19,6 +19,7 @@ def update_l10n(config, jobs):
 
         project = job["name"]
         repo_prefix = job.pop("repo-prefix")
+        pr_target = job.pop("pr-target", "master")
```

Comment:
maybe we should have the default here as "main" not "master"  (taskcluster/ci/update-project/kind.yml has a defaults of "master" anyway)